e2e/new/tests/build/testdata/Dockerfile

```diff
@@ -0,0 +1,34 @@
+################ Build & Dev ################
```

Comment:
While this works, I think we should use a simpler Dockerfile here to minimize test time.